### ISSUE TITLE
Main page scrolls when Learner opens info side panel in Bookmark page by keyboard navigation

### ISSUE THREAD

**Issue by sairina** [learningequality/kolibri#9095]
## Observed behavior
When a learner has bookmarked resources, if they use the keyboard to go to the Bookmark tab and open up the info side panel, they will see that as they navigate through the info side panel, the background from the main page scrolls.

## Errors and logs
![2022-02-09 22 34 38](https://user-images.githubusercontent.com/13563002/153351395-3a51f86e-564d-4958-8b1e-57696b3617d0.gif)

## Expected behavior
Ideally, the main content page will not move while a user has a side panel open.

## User-facing consequences
This is just a strange annoyance.

## Steps to reproduce
1. Sign in as learner
2. Bookmark a resource from the Library
3. Navigate to the Bookmark tab
4. Once in the Bookmarks, use the keyboard to navigate to a card, and to the info button icon
5. Press enter to open the info side panel
6. Notice that the main page will shift

## Context
Important: Kolibri v15.0.1

**Comment by Akash-Kumar-Sen** [comment]
Hi @sairina is this issue still open? Then I can give it a go.

**Comment by MisRob** [comment]
Hello @Akash-Kumar-Sen, thank you for your interest. This issue is open, and you're welcome to try to reproduce it and fix it. Please see [our developer's documentation ](https://kolibri-dev.readthedocs.io/en/develop/getting_started.html) if you haven't done so already. You should find everything needed to get started there.

Just note that on the `develop` branch, which is the branch mentioned in the documentation, it may not be currently possible to set up a new facility due to some work in progress (details can be found in [the comments of another issue](https://github.com/learningequality/kolibri/issues/9574)). So if you don't have Kolibri set up yet, please go through setup on the `release-v0.15.x` branch instead of `develop,` and it should be fine. So in the documentation, instead of
```
git checkout develop # Checkout the development branch
```
please do

```
git checkout release-v0.15.x
```

and then see if you can reproduce the problematic behavior reported in the issue. Its description says it should be present on the release branch.

**Comment by MisRob** [comment]
@Akash-Kumar-Sen If you could let me know as soon as you start working, it'd be great. I would assign you so that we know that the issue has been taken.

**Comment by Akash-Kumar-Sen** [comment]
Sure @MisRob you can assign this to me now.

**Comment by Akash-Kumar-Sen** [comment]
I have also showed interest in this issue : https://github.com/learningequality/kolibri/issues/9563 you can also assign that to me if it is still open.

**Comment by MisRob** [comment]
@Akash-Kumar-Sen Adding myself as an assignee too just to keep an eye on comments

**Comment by MisRob** [comment]
@Akash-Kumar-Sen Okay, you're also assigned to the other issue. Thank you.

**Comment by Akash-Kumar-Sen** [comment]
@MisRob Getting this error while running the step `yarn install` can you suggest what to do?
```
`(kolibri-4cBZohfZ) akash_sen@DESKTOP-ITNLNPR:~/kolibri$ yarn install
yarn install v1.22.19
$ node ./packages/kolibri-tools/lib/npm_deprecation_warning.js
[1/5] Validating package.json...
error kolibri-root@0.12.0-dev.1: The engine "node" is incompatible with this module. Expected version "10.x". Got "16.16.0"
error Found incompatible module.`
```

**Comment by MisRob** [comment]
@Akash-Kumar-Sen 

This asks you to use Node v10 (we haven't upgraded it to v16 on the current release branch)

```
nodeenv -p --node=10.13.0 
```

**Comment by amitpanwar789** [comment]
Hi @sairina is this issue still open? Then I can give it a go.

**Comment by deeppatel23** [comment]
Hi, if the issue is still open, I can give it a try. Can you assign it to me, @MisRob?

**Comment by MisRob** [comment]
Hello @deeppatel23, you're welcome to take this issue on. Please try if you can reproduce it at first. It'd be helpful if you could let us know as soon as you start working and I will assign you. If you haven't done so already, please have a look [at our developer documentation](https://kolibri-dev.readthedocs.io/en/develop/getting_started.html).

**Comment by deeppatel23** [comment]
Hi, I have done the setup and added a list of resources in Bookmarks. I am not able to product the issue, I can see that info (i) is clicked using the arrows of the keyboard. I am using Mac Keyboard, can you guide how to produce this issue?

**Comment by deeppatel23** [comment]
@MisRob any suggestion for this error when trying to install node=10.13.0: 
Please use either Python 2.6 or 2.7
nvm: install v10.13.0 failed!
I am on Mac M1 and my current branch if release-v0.15.x. I have checked for solutions on StackOverflow for this issue, but nothing works. 

Also, I tried to create this bookmark tab behaviour in develop branch, but the issue was not generated there.

**Comment by deeppatel23** [comment]
@MisRob the above-mentioned issue is fixed and I am able to reproduce this issue and I am working on it.

**Comment by MisRob** [comment]
Hi @deeppatel23, alright, thank you

**Comment by harshtalati2410** [comment]
Hello, @MisRob , I'm Harsh Talati. I would like to know if this issue is still open for contribution. If so, I would like to contribute to it. I've gone through the developers guide and if it is open, I can start working on it.

**Comment by MisRob** [comment]
Welcome @harshtalati2410. It seems no one is working on this issue, so you're welcome to take this on.

**Comment by Pursottam6003** [comment]
Hii @MisRob   I am interested in working on this issue. Could you please assign it to me so we can get started?
I can assure you that I will do this task within a few hours

**Comment by Pursottam6003** [comment]
![image](https://user-images.githubusercontent.com/81693090/229311031-56e1ba26-a96e-4fbf-b19a-1a6a5f483ee8.png)


Hi i had also tried to reproduce the issue but i am unable to figure out any shifts /srolls

**Comment by nikkuAg** [comment]
Hi, is this issue still open? I want to contribute to it.

**Comment by marcellamaki** [comment]
Hi @nikkuAg - this issue is still open, but has been open for quite some time and it's possible that it may have been fixed incidentally over development in the last year. You are welcome to work on it, if you are still able to reproduce it. If you're not able to, let us know. Thank you for your interest!

**Comment by nikkuAg** [comment]
Hi @marcellamaki, Yes I am able to reproduce this.

**Comment by nikkuAg** [comment]
@marcellamaki,
I have added a PR. Please review it.

**Comment by rtibbles** [comment]
Hi @nikkuAg - thanks for the fix, which I have merged - although it doesn't actually resolve _this_ issue which is specifically about the side panel in the bookmarks page. If you could replicate that same logic on the bookmark page, that would be great!

**Comment by rtibbles** [comment]
See here: https://github.com/learningequality/kolibri/blob/release-v0.16.x/kolibri/plugins/learn/assets/src/views/BookmarkPage.vue

**Comment by nikkuAg** [comment]
> Hi @nikkuAg - thanks for the fix, which I have merged - although it doesn't actually resolve _this_ issue which is specifically about the side panel in the bookmarks page. If you could replicate that same logic on the bookmark page, that would be great!

@rtibbles I have added a PR for the bookmarks page

**Comment by MisRob** [comment]
Unassigning due to long time of inactivity. See https://github.com/learningequality/kolibri/pull/11341#issuecomment-1805315723 for more

**Comment by MisRob** [comment]
Closed by https://github.com/learningequality/kolibri/pull/11607
